masterlist.yaml

```
@@ -27381,3 +27381,32 @@ plugins:
     tag:
       - Actors.AIData
       - Actors.AIPackages
+
+  - name: 'homeymealyum.esp'
+    url: [ 'https://www.nexusmods.com/skyrim/mods/87350' ]
+    tag: [Graphics]
+
+  - name: 'BeandBarbSpecials.esp'
+    url: [ 'https://www.nexusmods.com/skyrim/mods/77349' ]
+    tag:
+      - Graphics
+      - Invent.Change
+      - ObjectBounds
+
+  - name: 'Ahzidal''s Armor set - My version by Xtudo.esp'
+    url: [ 'https://www.nexusmods.com/skyrim/mods/108131' ]
+    tag: [Graphics]
+
+  - name: 'Ghostblade.esp'
+    url: [ 'https://www.nexusmods.com/skyrim/mods/92118' ]
+    tag: [Graphics]
+
+  - name: 'Stormfang.esp'
+    url: [ 'https://www.nexusmods.com/skyrim/mods/96905' ]
+    tag:
+      - Graphics
+      - Stats
+
+  - name: 'Horksbane.esp'
+    url: [ 'https://www.nexusmods.com/skyrim/mods/107114' ]
+    tag: [Graphics]
```

@Kaenguru-Knobel-Kette Just some small formatting changes for consistency, looks good otherwise:

```suggestion
  - name: 'homeymealyum.esp'
    url: [ 'https://www.nexusmods.com/skyrim/mods/87350/' ]
    tag: [ Graphics ]

  - name: 'BeandBarbSpecials.esp'
    url: [ 'https://www.nexusmods.com/skyrim/mods/77349/' ]
    tag:
      - Graphics
      - Invent.Change
      - ObjectBounds

  - name: 'Ahzidal''s Armor set - My version by Xtudo.esp'
    url: [ 'https://www.nexusmods.com/skyrim/mods/108131/' ]
    tag: [ Graphics ]

  - name: 'Ghostblade.esp'
    url: [ 'https://www.nexusmods.com/skyrim/mods/92118/' ]
    tag: [ Graphics ]

  - name: 'Stormfang.esp'
    url: [ 'https://www.nexusmods.com/skyrim/mods/96905/' ]
    tag:
      - Graphics
      - Stats

  - name: 'Horksbane.esp'
    url: [ 'https://www.nexusmods.com/skyrim/mods/107114/' ]
    tag: [ Graphics ]
```